### ISSUE TITLE
Screenshots

### ISSUE THREAD

**Issue by marco-trovato** [Poussinou/FLOSS-Games-on-Steam#34]
For the love of god add some screenshots!
Even lowres, as long as there is at least something that is not just a wall of text!

**Comment by marco-trovato** [comment]
(the list is awesome, that's why i care about this improvement)

**Comment by rammba** [comment]
Hey @ShiroTohu, I saw you started working on this :rocket:
Good job, for me it looks great with three images for every game. Do you plan to continue and add images for all other games as well? If you do, don't forget to fetch latest changes from this repo.

**Comment by ShiroTohu** [comment]
Yep. Adding images is tiresome and repetitive though so the branch has been in limbo for quite some time. I do plan on creating a script that makes it easier to add games since the games on the list follows a particular pattern but haven't gotten around to doing that.

**Comment by rammba** [comment]
I agree it's boring task. Good luck with that and I'm looking forward to see it done!
